lib/parsers.js

```diff
@@ -53,123 +53,105 @@ module.exports.parseError = function parseError(err, kwargs, cb) {
   });
 };
 
-module.exports.parseRequest = function parseRequest(req, kwargs) {
-  kwargs = kwargs || {};
+module.exports.parseRequest = function parseRequest(req, parseUser) {
```

Comment:
Is `parseRequest` a public/documented API method? If so, might want to consider how to make this signature more extendable going forward.

e.g. I'd almost prefer if we kept `kwargs` and you did:

```javascript
parseRequest(req, {
  parseUser: ['username']
});
```